Gems/GenAIFramework/Code/Include/GenAIFramework/Feature/AIAgentBus.h

```diff
@@ -31,9 +32,31 @@ namespace GenAIFramework
         //! The response will be sent to the OnAIResponse notification.
         virtual void SendPrompt(const AIMessages& prompt) = 0;
 
+        //! Send a prompt to the model agent as a JSON string.
+        //! @param prompt The prompt to send as a JSON string.
+        //! The prompt is a JSON string that is an array of objects. Each object consists of a role and content.
+        //! The role is a string that can be "user", "assistant", or "system". Corrsponding to the user, assistant, and system roles.
+        //! The content is an array of strings that are the content of the message.
+        virtual void SendPromptAsJsonString(const AZStd::string& prompt)
```

Comment:
I see a lack of consistency in the naming: `SendPromptAsJsonString` and `GetHistoryAsJson`. We could use the same name for sending: `SendPrompt` and keep `GetHistoryAsJson` as is. Or `SendTextPrompt` and `GetHistoryAsText`. Whatever :)

Reply:
I would keep the `SendPromptAsJsonString` name as O3DE EBus calling seems not to support function overloading.